src/common/envs/.env.example

```diff
@@ -6,4 +6,13 @@ DATABASE_HOST=
 DATABASE_NAME=
 DATABASE_USER=
 DATABASE_PASSWORD=
-DATABASE_PORT=
\ No newline at end of file
+DATABASE_PORT=
+
+
+TEST_DATABASE_HOST=
+TEST_DATABASE_NAME=
+TEST_DATABASE_USER=
+TEST_DATABASE_PASSWORD=
+TEST_DATABASE_PORT=
```

Comment:
Instead of leaving it up to the user, you can hard-code these values to make things a bit more streamlined.